docs/release-notes/2.20.0.md

```diff
@@ -0,0 +1,77 @@
+# Percona Monitoring and Management 2.20.0
+
+<table class="docutils field-list" frame="void" rules="none">
+  <colgroup>
+    <col class="field-name">
+    <col class="field-body">
+  </colgroup>
+  <tbody valign="top">
+    <tr class="field-odd field">
+      <th class="field-name">Date:</th>
+      <td class="field-body">August 3, 2021</td>
+    </tr>
+    <tr class="field-even field">
+      <th class="field-name">Installation:</th>
+      <td class="field-body">
+        <a class="reference external" href="https://www.percona.com/software/pmm/quickstart">Installing Percona Monitoring and Management</a></td>
+    </tr>
+  </tbody>
+</table>
+
+Percona Monitoring and Management (PMM) is a free and open-source platform for managing and monitoring MySQL, MongoDB, and PostgreSQL performance.
+
+## Release Highlights
+
+- The [Easy-install script] for PMM Server is checksum verified.
+- PMM will use [Grafana 7.5] instead of 7.3. We are also preparing for Grafana 8.
+- PostgreSQL monitoring with the `pg_stat_monitor` plugin enabled exposes new metrics such as Plan Times, WAL Fpi/Bytes/Records.
+- For users who deploy PMM Server through the AWS Marketplace, AWS RDS service discovery will be executed without AWS credentials and tuning [IAM roles].
+- For Backup Management (Technical Preview), we added the ability to schedule backups so you can schedule and see already scheduled [backups] in the UI.
+
+## Important note for users of PMM 2.16.0 via Docker
```

Comment:
```suggestion
## Important note for users of PMM via Docker who started with version 2.16.0
```

Reply:
@nikita-b There's a lot of info to pack into this title:
- User uses Docker..
- .. started with version 2.16
- ..either still be on that version, or
- have upgraded from it to any one of 2.17, 2.28, 2.19 or 2.20.
Do you think the title is enough?